backend/open_webui/models/chats.py

```
@@ -6,12 +6,14 @@
 
 from open_webui.internal.db import Base, get_db
 from open_webui.models.tags import TagModel, Tag, Tags
+from open_webui.models.folders import Folders
 from open_webui.env import SRC_LOG_LEVELS
 
 from pydantic import BaseModel, ConfigDict
 from sqlalchemy import BigInteger, Boolean, Column, String, Text, JSON
 from sqlalchemy import or_, func, select, and_, text
```

Ruff (0.12.2)

`sqlalchemy.func` imported but unused

Remove unused import

(F401)